### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by shashisp**

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      };
+    } catch (e) {
+      return {
+        type: DELETE_USER,
+        payload: user
+      } ;
+    }
+  };
+
+  let previousId = 4;
```
Comment (adamstep) [inline src/actions/index.js]: this previousId seems like a bit of a hack. Is it possible to derive from the state?

**Comment by adamstep** [comment]
@shashisp I saw one weird thing when testing. After I add a user, if I click them then the page reloads with the user gone. Can you look into it?

**Comment by shashisp** [comment]
> @shashisp I saw one weird thing when testing. After I add a user, if I click them, then the page reloads with the user gone. Can you look into it?


@adamstep  not sure if I understood this correctly  https://cl.ly/5e04260517d8 we're not storing the data, the initial state is being rendered from the hardcoded list https://github.com/shashisp/teams-frontend/pull/1/files#diff-cfba9c145ca5d713b626b9c58bb518a7R5  after refresh it goes back to the initial state

File: src/actions/index.js (hunk 1 above)
Comment (shashisp) [reply src/actions/index.js]: Sure, I'll update the id based on the last available user object in the list.

**Comment by adamstep** [comment]
@shashisp I understand the data is only stored locally. But why does clicking on an added item refresh the page, as opposed to showing the edit state (without a page reload)?

**Comment by shashisp** [comment]
> @shashisp I understand the data is only stored locally. But why does clicking on an added item refresh the page, as opposed to showing the edit state (without a page reload)?

@adamstep sorry, I hadn't noticed that issue got confused, only name data was wrapped under the link, now I've fixed it.

**Pull request merged**